DESCRIPTION

```diff
@@ -1,22 +1,23 @@
 Package: statbotData
 Title: What the Package Does (One Line, Title Case)
 Version: 0.0.0.9000
-Authors@R: 
+Authors@R:
     person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
            comment = c(ORCID = "YOUR-ORCID-ID"))
 Description: What the package does (one paragraph).
 License: MIT + file LICENSE
-Suggests: 
+Suggests:
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
-Imports: 
+Imports:
     assertthat,
     BFS,
     dplyr,
     googlesheets4,
     magrittr,
+    stringdist,
```

Comment:
I couldn't install this package. On google I found this: https://github.com/markvanderloo/stringdist/issues/84
This shouldn't prevent us from using it though. I just wanted to mention this.

Reply:
Based on this comment https://github.com/velocyto-team/velocyto.R/issues/57#issuecomment-506671728, it looks like an issue with llvm+R on MacOS.